src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -109,14 +121,36 @@ class HoldConfirmation extends React.Component {
     return (
       <Link
         className="nypl-request-button"
-        to="/"
+        to={`${appConfig.baseUrl}/`}
         onClick={(e) => this.goRestart(e)}
       >
         Start Over
       </Link>
     );
   }
 
+  /**
+   * renderBackToSearchLink
+   * Renders the link back to the page of search results.
+   *
+   * @return {HTML Element}
+   */
+  renderBackToSearchLink() {
+    if (!this.props.location.query.searchKeywords) {
```

Comment:
So the reason why is for no js. If there's no js, we have to rely on the url's query to get the search keywords.